packages/core-utils/src/query-params.js

```diff
@@ -176,7 +176,7 @@ const queryParams = [
       query.mode &&
       hasTransit(query.mode) &&
       query.mode.indexOf("WALK") !== -1,
-    default: 1207, // 3/4 mi.
+    default: 1609, // 1 mi.
```

Comment:
This change will affect existing OTP-RR deployments where the default walk distance is 3/4 mile. You can change the default walking distance to be specific to you implementation if you use `getCustomQueryParams` from `core-utils/query-params` with your overrides. Please see https://github.com/opentripplanner/otp-ui/blob/b3260af2279671825167cc35f36f56da5de0eebd/packages/core-utils/src/query-params.js#L911

Reply:
Binh, using that creaky API call won't work for us (already looked).  I'm sure your implementations can change to 1 mile as a default, and not miss a beat.  This is a library that TriMet funded, and we want 1 mile as the default.  If IBI wants to go the extra distance to make this configurable (https://github.com/opentripplanner/otp-ui/issues/373), that's up to you.

Reply:
Hi @fpurcell, IBI has the ability to customize the default walk distance that way (but we'd have to make changes to OTP-RR after this PR is merged), so I'm fine to approve.